core/vm/contracts_suave_runtime_adapter.go

```diff
@@ -42,14 +47,18 @@ var (
 	fetchBidsAddr                = common.HexToAddress("0x0000000000000000000000000000000042030001")
 	fillMevShareBundleAddr       = common.HexToAddress("0x0000000000000000000000000000000043200001")
 	newBidAddr                   = common.HexToAddress("0x0000000000000000000000000000000042030000")
+	randomUintAddr               = common.HexToAddress("0x00000000000000000000000000000000F00bA777")
```

Comment:
should the hex addresses follow the same scheme as the others?

Reply:
I see a loose pattern in how the addresses are chosen, but it doesn't seem totally consistent.
For example, from 
```go
fetchBidsAddr                = common.HexToAddress("0x0000000000000000000000000000000042030001")
```
we might infer that `0x00..42` implies MEV-Share bid-related precompiles. But then we see
```go
fillMevShareBundleAddr       = common.HexToAddress("0x0000000000000000000000000000000043200001")
```
So then we might infer that `0x00..42` is strictly bids and `0x00..43` is just bundle-related. But then we see
```go
simulateBundleAddr           = common.HexToAddress("0x0000000000000000000000000000000042100000")
```
So then my guess is that the scheme is "start with 0x000000000000000000000000000000004 and have some zeros in the middle?"... Is there an actual pattern I should be following?
I thought this address was fun :p but happy to change it -- just wondering how best to avoid conflicts w/ other precompile PRs.